rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1470,21 +1472,25 @@ protected void doCREATE(){
         program.step();
     }
 
+    private DataWord getCodeSize() {
+        return stack.get(stack.size() - 3);
+    }
+
     protected void doCREATE2(){
         if (program.isStaticCall()) {
             throw Program.ExceptionHelper.modificationException(program);
         }
 
         if (computeGas){
-            long codeSize = stack.get(stack.size() - 3).longValueSafe();
+            DataWord codeSize = getCodeSize();
```

Comment:
not sure it's beneficial to make such refactoring - I mean extracting `stack.get(stack.size() - 3)` into a method. As for me, it was more explicit which item of stack was being extracted. Also `codeSize.longValue()`, which is used in several places below does some internal calcs

Reply:
Well, in my opinion it worth to to the refactor from getCodeSize because it's more clear that you are getting the code size from the stack. Do you see a problem in re-use this logic?

The thing with the long value it's because I need the intValue in the call. So I would have to get this value again converted, if I get the object that can be used for both situations isn't better?